src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
 // export const Person = ({ person }) => ();
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+    {person.age !== undefined && (<p className="Person__age">{`I am ${person.age}`}</p>)}
+
+    <p className="Person__partner">
+      {person.isMarried === false && 'I am not married'}
+      {(person.isMarried === true && person.sex === 'm') && `${person.partnerName} is my wife`}
+      {(person.isMarried === true && person.sex === 'f') && `${person.partnerName} is my husband`}
```

Comment:
try to simplify it using the ternary operator and decide before jsx what text will be rendered and just render in jsx. 

something like that
```
const personPartnerType = sex === 'm' ? 'wife' : 'husband';
const personPartner = isMarried ? `${partnerName} is my ${personPartnerType}` : 'I am not married'

```